sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/_models.py

```diff
@@ -768,12 +770,13 @@ def _to_certificate_policy_bundle(self):
                 key_size=self.key_size,
                 reuse_key=self.reuse_key,
                 curve=self.key_curve_name,
-            )
+            )  # type: Optional[models.KeyProperties]
         else:
             key_properties = None
 
         if self.content_type:
-            secret_properties = models.SecretProperties(content_type=self.content_type)
+            secret_properties = models.SecretProperties(
+                content_type=self.content_type)  # type: Optional[models.SecretProperties]
```

Comment:
nit: I think the parenthesis after `self.content_type` should be on the next line -- did you try running `black` on this file?

Reply:
Nope. But I see some other places are putting end parenthesis in a new line, so will follow with others.